### ISSUE TITLE
Azure Function Doesn't work

### ISSUE THREAD

**Issue by mswilson4040** [Microsoft/azure-pipelines-tasks#9921]
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Bug

**Enter Task Name**: Azure Function  

list here (V# not needed):  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

## Environment
- Server - Azure Pipelines or TFS on-premises?
    Azure Pipelines
    - If using TFS on-premises, provide the version: 
    
    - If using Azure Pipelines, provide the account name, team project name, build definition name/build number: 

- Agent - Hosted or Private: 
    
    - If using Hosted agent, provide agent queue name:

    - If using private agent, provide the OS of the machine running the agent and the agent version: 

## Issue Description

[Include task name(s), screenshots and any other relevant details]

The Azure Function task does not work when deploying an Azure Function. When the function is deployed, the Azure Portal won't show you the functions (it doesn't list them at all). If you try and hit the function url, you just get a 404. If you go back and deploy directly from Visual Studio or Visual Studio Code, things start to work.

### Task logs

[Enable debug logging and please provide the zip file containing all the logs for a speedy resolution]

## Troubleshooting
Checkout how to troubleshoot failures and collect debug logs: https://docs.microsoft.com/en-us/vsts/build-release/actions/troubleshooting

### Error logs

[Insert error from the logs here for a quick overview]

**Comment by niadak** [comment]
@mswilson4040, Thanks for reporting the issue. Can you please share the debug logs with us. Please follow the [instruction ](https://docs.microsoft.com/en-us/azure/devops/pipelines/troubleshooting?view=azure-devops#get-logs-to-diagnose-problems)to generate debug logs.

**Comment by mswilson4040** [comment]
@niadak. Apologies as I forgot to update this. As it turns out, the entire issue was due to me sending the wrong build path in azure devops to package the functions. Once I corrected the file path, it worked without issue. My only feedback at this point would be to improve the messaging there. The error/information provided didn't help me get to the core issue at all. It look a few days to realize it was just a stupid mistake.
